jwql/website/apps/jwql/templates/api_landing.html

```diff
@@ -0,0 +1,152 @@
+                                    'jw97012001001_02101_00001_guider1_rateints_integ0.thumb',
+                                    'jw97012001004_02101_00001_guider1_rate_integ0.thumb',
+                                    'jw97012001004_02101_00001_guider1_trapsfilled_integ0.thumb',
+                                    'jw97012001001_02101_00001_guider1_trapsfilled_integ1.thumb',
+                                    'jw97012001004_02101_00001_guider1_trapsfilled_integ1.thumb',
+                                    ...,
+                                    ...]}
+                        </code>
+                    </pre>
+
+                    <h5>Obtaining Filenames Associated with Proposal</h5>
+                    <p align="justify">
+                        A user is interested in all of the thumbnail associated with the Fine Guider Sensors (FGS).  
+                    </p>
+                    <pre>
+                        <code class="python">
+                        from urllib import request
+                        import json
+    
+                        # Build the URL https://dljwql.stsci.edu/api/proposal/service/
+                        url = 'https://dljwql.stsci.edu/api/86600/filenames/'
+                        r = request.urlopen(url) 
+                        filenames = json.loads(r.read().decode())
+    
+                        print(filenames)
+    
+                        {'filenames': ['jw86600001001_02101_00001_guider1_cal.fits',
+                                       'jw86600001001_02101_00001_guider1_rate.fits',
+                                       'jw86600001001_02101_00001_guider1_trapsfilled.fits',
+                                       'jw86600001001_02101_00001_guider1_uncal.fits',
+                                       'jw86600001001_02101_00002_guider1_cal.fits',
+                                       'jw86600001001_02101_00002_guider1_rate.fits',
+                                       'jw86600001001_02101_00002_guider1_trapsfilled.fits',
+                                       'jw86600001001_02101_00002_guider1_uncal.fits',
+                                       'jw86600001001_02101_00003_guider1_cal.fits',
+                                       'jw86600001001_02101_00003_guider1_rate.fits',
+                                       'jw86600001001_02101_00003_guider1_trapsfilled.fits',
+                                       'jw86600001001_02101_00003_guider1_uncal.fits',
+                                       'jw86600001001_02101_00004_guider1_cal.fits',
+                                       'jw86600001001_02101_00004_guider1_rate.fits',
+                                       'jw86600001001_02101_00004_guider1_trapsfilled.fits',
+                                       ...,
+                                       ...]}
+                        </code>
+                    </pre>
+        </div>
+
+        <div id="api_list">
+            <h4>List of Available Services</h4><hr>
```

Comment:
It would be nice if the items in this list had an accompanying example URL, something like:

```
Instrument Proposals (e.g. https://dljwql.stsci.edu/api/nirspec/proposals/)
```

Reply:
Do you think the text "Instrument Proposals" be a like or should we include the text with the link next to it? i.e. `service: (https://dljwql.stsci.edu/api/instrument/service)`?

Reply:
Oh, I see what you mean! I am on it

Reply:
Okay done!